src/svg/Painter.ts

```diff
@@ -108,6 +108,7 @@ class SVGPainter implements PainterBase {
             vnode.attrs.style = 'user-select:none;position:absolute;left:0;top:0;';
             patch(this._oldVNode || this._svgDom, vnode);
             this._oldVNode = vnode;
+            this._svgDom = vnode.elm as SVGElement;
```

Comment:
In which case the svg dom will be changed? 

I'm concerning if it will cause bug if it's changed after top code getting it through `getSvgDom`

Reply:
Made an online demo for this. Please refer to https://codepen.io/plainheart/pen/BawBeMx.

Reply:
Seems it's not expected. Take a deep look at, seems it's because if we doing the first patch and using the container as patch target instead of a vnode in
https://github.com/ecomfe/zrender/blob/9fb1577705ac21e15deb4fc8562a5ca907398cb4/src/svg/Painter.ts#L109

It will force create an empty vnode for this container in https://github.com/ecomfe/zrender/blob/9fb1577705ac21e15deb4fc8562a5ca907398cb4/src/svg/patch.ts#L330

Which has a different `key` with the new vnode and cause elm recreated. Perhaps we create the `_oldVNode` ahead when svgDom is created.